corehq/apps/app_manager/helpers/validators.py

```diff
@@ -899,7 +899,7 @@ def check_actions(self):
         from corehq.apps.app_manager.models import AdvancedOpenCaseAction, LoadUpdateAction
 
         for action in self.form.actions.get_subcase_actions():
-            case_tags = self.form.actions.get_case_tags()
+            case_tags = list(self.form.actions.get_case_tags())
```

Comment:
should this be outside of the for loop so that its not recalculated for every action?

Reply:
good point, 88def0fbd20